dev/packages/example/cisco-1.0.0/dataset/asa/filebeat/config/input.yml

```diff
@@ -0,0 +1,15 @@
+{{#if input == "syslog"}}
```

Comment:
This will become very interesting as it actually uses 2 different inputs which we support in our manifest but did not have an example for it yet. I think this will have to go into 2 different files.

I wonder why you keep the filebeat config inside?